### ISSUE TITLE
Index was out of range when saving file

### ISSUE THREAD

**Issue by irvalchev** [EPPlusSoftware/EPPlus#381]
I receive the following error while saving a file which was previously edited with EPPlus. Unfortunately I cannot share a reproducible Excel.

```
System.ArgumentOutOfRangeException
  HResult=0x80131502
  Message=Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
  Source=System.Private.CoreLib
  StackTrace:
   at System.ThrowHelper.ThrowArgumentOutOfRange_IndexException()
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at OfficeOpenXml.ExcelStyleCollection`1.get_Item(Int32 PositionID) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\ExcelStyleCollection.cs:line 94
   at OfficeOpenXml.Drawing.ExcelDrawing.GetRowHeightFromCellFonts(Int32 row, ExcelWorksheet ws) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawing.cs:line 768
   at OfficeOpenXml.Drawing.ExcelDrawing.GetRowHeight(Int32 row) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawing.cs:line 745
   at OfficeOpenXml.Drawing.ExcelDrawing.GetPixelHeight() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawing.cs:line 704
   at OfficeOpenXml.Drawing.ExcelDrawing.GetPositionSize() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawing.cs:line 1278
   at OfficeOpenXml.Drawing.ExcelDrawing..ctor(ExcelDrawings drawings, XmlNode node, String topPath, String nvPrPath, ExcelGroupShape parent) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawing.cs:line 88
   at OfficeOpenXml.Drawing.ExcelShapeBase..ctor(ExcelDrawings drawings, XmlNode node, String topPath, String nvPrPath, ExcelGroupShape parent) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelShapeBase.cs:line 48
   at OfficeOpenXml.Drawing.ExcelShape..ctor(ExcelDrawings drawings, XmlNode node, ExcelGroupShape shape) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelShape.cs:line 24
   at OfficeOpenXml.Drawing.ExcelDrawing.GetDrawingFromNode(ExcelDrawings drawings, XmlNode node, XmlElement drawNode, ExcelGroupShape parent) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawing.cs:line 565
   at OfficeOpenXml.Drawing.ExcelDrawing.GetDrawing(ExcelDrawings drawings, XmlNode node) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawing.cs:line 549
   at OfficeOpenXml.Drawing.ExcelDrawings.AddDrawings() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawings.cs:line 118
   at OfficeOpenXml.Drawing.ExcelDrawings..ctor(ExcelPackage xlPackage, ExcelWorksheet sheet) in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\Drawing\ExcelDrawings.cs:line 89
   at OfficeOpenXml.ExcelWorksheet.get_Drawings() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\ExcelWorksheet.cs:line 3583
   at OfficeOpenXml.ExcelWorksheet.SaveDrawings() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\ExcelWorksheet.cs:line 2418
   at OfficeOpenXml.ExcelWorksheet.Save() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\ExcelWorksheet.cs:line 2381
   at OfficeOpenXml.ExcelWorkbook.Save() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\ExcelWorkbook.cs:line 1128
   at OfficeOpenXml.ExcelPackage.Save() in C:\Users\Iliya\Downloads\EPPlus-develop\src\EPPlus\ExcelPackage.cs:line 843
```

Some additional debug info:
![image](https://user-images.githubusercontent.com/2835895/116215384-eae00a80-a747-11eb-9b75-5c24d703951d.png)
![image](https://user-images.githubusercontent.com/2835895/116215891-5e821780-a748-11eb-8eac-5269935ed94e.png)
Strangely enough, when I see the Value from cse (same debug session), the Value is update to (seemingly) the proper value
![image](https://user-images.githubusercontent.com/2835895/116216123-90937980-a748-11eb-8966-8896510fd5e0.png)

**Comment by JanKallman** [comment]
Hard to tell what goes wrong here. I will need a reproducible workbook to find this issue

**Comment by irvalchev** [comment]
Here a file to reproduce it [safe_three_stage_LED_EPPlusErr.Standalone.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6398338/safe_three_stage_LED_EPPlusErr.Standalone.xlsx). You should be able to reproduce it, by making any changes and trying to save. Or just by accessing the Drawings of the second sheet (Overview).
Once the file is edited and saved in Excel the issue disappears (it seems Excel automatically detects and fixes the issue).

**Comment by JanKallman** [comment]
Thanks, 
The problem seems to be that the styleId (s attribute) is set to int.MinValue in the template you supply...
For example:
```Xml
<row r="3" s="-2147483648" ht="15.75" customFormat="1">
  <c r="B3" s="5" t="s">
    <v>27</v>
   </c>
 ...
</row>`
```
Is the supplied template also created in the latest version of EPPlus?

**Comment by JanKallman** [comment]
Fixed in 5.6.4
